app/src/main/java/org/simple/clinic/patient/businessid/Identifier.kt

```diff
@@ -40,7 +40,15 @@ data class Identifier(
       }
       BangladeshNationalId -> value
       EthiopiaMedicalRecordNumber -> value
-      IndiaNationalHealthId -> value
+      IndiaNationalHealthId -> {
+        val enteredCode = IndiaNationalHealthId.enteredCodeLength(this)
+        val prefix = enteredCode.substring(0, 2)
+        val subString1 = enteredCode.substring(2,6)
+        val subString2 = enteredCode.substring(6,10)
+        val suffix = enteredCode.substring(10)
+
+        "$prefix${Unicode.nonBreakingSpace}$subString1${Unicode.nonBreakingSpace}$subString2${Unicode.nonBreakingSpace}$suffix"
```

Comment:
Did we update the `IdentifierDisplayTest` to verify that we are getting the correct string here?

Reply:
That unit test is failing, so do update that.